backend/core/constants.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+EMAIL_FROM = os.getenv("EMAIL_FROM", "")
```

Comment:
What does constants.py do differently than settings.py?

Reply:
What does django do with this variable/ how does it find it?

Reply:
it looks cooler